src/osgEarthSymbology/TextSymbol.cpp

```diff
@@ -83,7 +84,8 @@ TextSymbol::TextSymbol( const Config& conf ) :
                                              _attachedLabelDecimationTolerance( false ),
                                              _minRange             ( DBL_MAX ),
                                              _minRange2ndlevel     ( DBL_MAX ),
-                                             _predefinedOrganisationMargin( 4. )
+                                             _predefinedOrganisationMargin( 4. ),
+                                             _textPriority         ( 0.0 )
```

Comment:
**Impossible to understand the difference between TextSymbol::priority() and TextSymbol::textPriority**

replace textPriority by priorityFine everywhere